Description: raises the self-test timeout to 4 minutes for really slow machines.
Author: Kees Cook <kees@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=699774

--- a/parser/tst/simple.pl
+++ b/parser/tst/simple.pl
@@ -10,7 +10,7 @@
 my %config;
 $config{'parser'} = "/sbin/apparmor_parser";
 $config{'profiledir'} = "./simple_tests/";
-$config{'timeout'} = 120; # in seconds
+$config{'timeout'} = 480; # in seconds
 
 my $help;
 my $pwd = `pwd`;
